src/DotNetty.Common/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Nito.Deque" version="1.0.0" targetFramework="net45" />
```

Comment:
So inline with the previous question then, this NuGet package just adds a source file and no actual binary?